Wireframe/index.html

```diff
@@ -4,10 +4,74 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Wireframe</title>
-    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="css/style.css">
   </head>
   <body>
     <!-- Add your HTML markup here -->
     <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
-</body>
+      <header>
+        <h1>Introduction</h1>
+        <p> Git, What is a branch in Git? and Why do developers need Git?</p>
+      </header>
+
+    <main>
+        <section class="first__paragraph">
+          <article class="layout"></article>
+        <article class="art__content">
+            <h1 class= "first__title">What is Git?</h1>
+            <p class="first__summary">Git is a version control system that helps you 
+              track changes in your files and collaborate with others on software 
+              development projects. It allows you to save 
+              different versions of your work and manage them efficiently.
+          </p>
+    
+          <p><a href="#"> Learn more</a></p>
+           
+          </article>
+           <hr class="new" />
+      </section>
+     
+      <section class="text__section">
+        <article class="art__main">
+           <img
+            class="article__img"
+            src="https://t3.ftcdn.net/jpg/02/61/85/92/360_F_261859293_WvBhDNgsf1H58DCUcrAyHqPXvoEu8cGV.jpg"
+            alt="to be creative"
+          />
+          <h1 class="art__title">What is a branch in Git?</h1>
+          <p class="art__summary">In Git, a branch is a separate line of development
+             that allows you to work on new features or fixes without affecting the main codebase. 
+            It's like a parallel workspace for making changes, which can later be merged back into the main code.
+          them.</p>
+            
+               <p><a href="#"> Learn more</a></p>
+           </article>
+          
+
+        <article class="art__main">
+           <img
+            class="article__img"
```

Comment:
Great job on using descriptive class names, @BakhatBegum ! To enhance your naming further, consider adopting the BEM (Block Element Modifier) naming convention. A more structured approach to class naming. You can learn more about BEM [here](https://getbem.com/naming/). Keep up the good work!